<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogonGroupEditor</class>
 <widget class="QDialog" name="LogonGroupEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Logon group editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Please add the groups whose members should be allowed to access computers in your iTALC network.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>431</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>All groups</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="addLogonGroup">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../imc.qrc">
         <normaloff>:/Resources/go-next.png</normaloff>:/Resources/go-next.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="removeLogonGroup">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../imc.qrc">
         <normaloff>:/Resources/go-previous.png</normaloff>:/Resources/go-previous.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Groups with iTALC privileges</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QListWidget" name="allGroups">
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QListWidget" name="logonGroups">
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../imc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LogonGroupEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LogonGroupEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addLogonGroup</sender>
   <signal>clicked()</signal>
   <receiver>LogonGroupEditor</receiver>
   <slot>addLogonGroup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeLogonGroup</sender>
   <signal>clicked()</signal>
   <receiver>LogonGroupEditor</receiver>
   <slot>removeLogonGroup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addLogonGroup()</slot>
  <slot>removeLogonGroup()</slot>
 </slots>
</ui>
